libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -342,7 +358,7 @@ AP_GPS_UBLOX::read(void)
                 _delay_time = 750;
             }
         } else {
-            _delay_time = 2000;
+            _delay_time = 4000;
```

Comment:
Is there a better mavlink message for bringing this to the autopilot? (And if not should we make one?) Or will it simply always be to big to fit in a mavlink1 packet?

I'm just looking for a way to not risk this type of collision/fracturing, and it would be nice if we could handle this in the driver somehow.